packages/muelu/src/Utils/ExplicitInstantiation/ETI_SC_LO_GO_NO_classes.cmake

```diff
@@ -73,7 +73,7 @@ APPEND_SET(MUELU_SC_LO_GO_NO_ETI_CLASSES MueLu::RebalanceBlockAcFactory )
 APPEND_SET(MUELU_SC_LO_GO_NO_ETI_CLASSES MueLu::RebalanceBlockInterpolationFactory )
 APPEND_SET(MUELU_SC_LO_GO_NO_ETI_CLASSES MueLu::RebalanceBlockRestrictionFactory )
 APPEND_SET(MUELU_SC_LO_GO_NO_ETI_CLASSES MueLu::RebalanceTransferFactory )
-APPEND_SET(MUELU_SC_LO_GO_NO_ETI_CLASSES MueLu::RegionRFactory )
+APPEND_SET(MUELU_SC_LO_GO_NO_ETI_CLASSES MueLu::RegionRFactory-.?if.defined[HAVE_MUELU_KOKKOS_REFACTOR] )
```

Comment:
Do we need the Kokkos refactor branch for that or just Tpetra to be enabled?

Reply:
`HAVE_MUELU_TPETRA` seems to suffice.

Reply:
After double checking we need Kokkos for sure here. Sorry for the misleading question above.

Reply:
Done.